### ISSUE TITLE
Some console error spam, but the plugin is working fine.

### ISSUE THREAD

**Issue by mistermodcreator** [cjburkey01/ClaimChunk#214]
**Versions**
Spigot Version: Paper-243
ClaimChunk Version: 0.0.22

**Describe the bug**
Plugin spams the following error in the console:

```
[11:23:07 WARN]: java.lang.ClassNotFoundException: net.minecraft.server.v1_17_R1.PacketPlayOutTitle
[11:23:07 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:146)
[11:23:07 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:103)
[11:23:07 WARN]:        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:519)
[11:23:07 WARN]:        at java.base/java.lang.Class.forName0(Native Method)
[11:23:07 WARN]:        at java.base/java.lang.Class.forName(Class.java:375)
[11:23:07 WARN]:        at claimchunk-0.0.22.jar//com.cjburkey.claimchunk.packet.PacketHandler.getNMSClass(PacketHandler.java:32)
[11:23:07 WARN]:        at claimchunk-0.0.22.jar//com.cjburkey.claimchunk.packet.TitleHandler.showTitle(TitleHandler.java:78)
[11:23:07 WARN]:        at claimchunk-0.0.22.jar//com.cjburkey.claimchunk.packet.TitleHandler.showTitle(TitleHandler.java:31)
[11:23:07 WARN]:        at claimchunk-0.0.22.jar//com.cjburkey.claimchunk.Utils.toPlayer(Utils.java:58)
[11:23:07 WARN]:        at claimchunk-0.0.22.jar//com.cjburkey.claimchunk.event.PlayerMovementHandler.showTitleRaw(PlayerMovementHandler.java:111)
[11:23:07 WARN]:        at claimchunk-0.0.22.jar//com.cjburkey.claimchunk.event.PlayerMovementHandler.showTitle(PlayerMovementHandler.java:90)
[11:23:07 WARN]:        at claimchunk-0.0.22.jar//com.cjburkey.claimchunk.event.PlayerMovementHandler.onPlayerMove(PlayerMovementHandler.java:60)
[11:23:07 WARN]:        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor59.execute(Unknown Source)
[11:23:07 WARN]:        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69)
[11:23:07 WARN]:        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
[11:23:07 WARN]:        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[11:23:07 WARN]:        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:624)
[11:23:07 WARN]:        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1477)
[11:23:07 WARN]:        at net.minecraft.network.protocol.game.PacketPlayInFlying.a(PacketPlayInFlying.java:114)
[11:23:07 WARN]:        at net.minecraft.network.protocol.game.PacketPlayInFlying$PacketPlayInPositionLook.a(PacketPlayInFlying.java:16)
[11:23:07 WARN]:        at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$ensureRunningOnSameThread$1(PlayerConnectionUtils.java:56)
[11:23:07 WARN]:        at net.minecraft.server.TickTask.run(TickTask.java:18)
[11:23:07 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:149)
[11:23:07 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.executeTask(IAsyncTaskHandlerReentrant.java:23)
[11:23:07 WARN]:        at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:1418)
[11:23:07 WARN]:        at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:192)
[11:23:07 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:122)
[11:23:07 WARN]:        at net.minecraft.server.MinecraftServer.bf(MinecraftServer.java:1396)
[11:23:07 WARN]:        at net.minecraft.server.MinecraftServer.executeNext(MinecraftServer.java:1389)
[11:23:07 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:132)
[11:23:07 WARN]:        at net.minecraft.server.MinecraftServer.sleepForTick(MinecraftServer.java:1367)
[11:23:07 WARN]:        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1278)
[11:23:07 WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319)
[11:23:07 WARN]:        at java.base/java.lang.Thread.run(Thread.java:831)

```
**To Reproduce**
Don't know how i got this error.

**Expected behavior**
Probably an update fixing this? Or some help if i can fix it myself.

**Additional context**
I already tried reinstalling the plugin.

**Comment by Geolykt** [comment]
known (and already solved) bug, duplicate of #208

**Comment by mistermodcreator** [comment]
Didn't know that. Thanks.

**Comment by cjburkey01** [comment]
> 
> 
> Didn't know that. Thanks.

If you would like to download a version that includes this fix, here's a preview of 0.0.23 (if you used custom protection settings in the config, they won't be loaded and you'll have to open up the new worlds folder to configure the world permissions).

[ClaimChunk.zip](https://github.com/cjburkey01/ClaimChunk/files/7110203/ClaimChunk.zip)
